### PR TITLE
Improve dutch translations

### PR DESCRIPTION
**Pull request by jvanbaarsen**
A couple translations were missing from the translation file. This adds a couple
of missing entries

### CODE REVIEW

**Comment by jvanbaarsen** [comment]
@dwradcliffe Is there anything I can do to have this merged?

File: config/locales/nl.yml [hunk 1]
```diff
@@ -49,9 +49,9 @@ nl:
     create:
       promise_resend:
     new:
-      submit:
-      title:
-      will_email_notice:
+      submit: Bevestiging opnieuw versturen
+      title: Bevestigingsemail opnieuw sturen
+      will_email_notice: We zullen een nieuwe email sturen met een bevestigings link
```
Comment (maartenvg) [inline config/locales/nl.yml]: `We zullen je een nieuwe email sturen met een bevestigingslink` (added `je` and `bevestigingslink` is one word)

Comment (jvanbaarsen) [inline config/locales/nl.yml]: The long words in dutch always keeps triping me up xD Thanks! Will update later today!

**Comment by jvanbaarsen** [comment]
@dwradcliffe I think this can be merged when tests are green :-)

**Comment by dwradcliffe** [comment]
Thanks!

**Pull request merged**